release/charts/charts.go

```diff
@@ -93,3 +190,47 @@ func runChartsBuild(chartsRepoPath string, args ...string) ([]byte, error) {
 
 	return output, nil
 }
+
+// debugPullRequest will prompt the user to check the files and commits that will be pushed to the remote repository
+func debugPullRequest(r *git.Repository, remote, b string) error {
+	var (
+		execute bool
+	)
+
+	if execute = ecmExec.UserInput("Check files that will be pushed?"); execute {
+		// commit history
+		iter, err := r.Log(&git.LogOptions{})
+		if err != nil {
+			return err
+		}
+
+		err = iter.ForEach(func(c *object.Commit) error {
+			fileStats, err := c.Stats()
+			if err != nil {
+				fmt.Println(err)
+				os.Exit(1)
+			}
+
+			fmt.Println(c)
+			fmt.Println(fileStats)
+
+			if execute = ecmExec.UserInput("Check next files that will be pushed?"); !execute {
```

Comment:
I assume there is an error from the CLI exec when this is ran. Where does that end up?